localization/english/culture/wc_culture_titles_l_english.yml

```diff
@@ -1,43 +1,344 @@
+ grand_mada_female_blood_troll:0 "Grand Ma'da"
+ queen_mada_female_blood_troll:0 "Queen Ma'da"
+ divine_mada_female_blood_troll:0 "Divine Ma'da" 
  
- lich_king_male:0 "Lich King"
- lich_king_female:0 "Lich Queen"
+ ### Drakkari ###
+ # Feudal
+ frost_king_feudal:0 "Frost King"
+ frost_queen_feudal:0 "Frost Queen"
+
+ ### Dragons ###
+ overlord_male_dragon:0 "Overlord"
+ overlady_female_dragon:0 "Overlady"
+ lord_male_dragon:0 "Lord"
+ lady_female_dragon:0 "Lady"
+ dragonlord_male_dragon:0 "Dragon Lord"
+ dragonlady_female_dragon:0 "Dragon Lady"
+
+ prince_male_dragon_kingdom:0 "Prince"
+ princess_female_dragon_kingdom:0 "Princess"
+ prince_male_dragon_empire:0 "Prince"
+ princess_female_dragon_empire:0 "Princess"
+
+ ### Red Dragons ###
+ dragonking_male_red_dragon:0 "Dragon King"
+ dragonqueen_female_red_dragon:0 "Dragon Queen"
+ dragonprince_male_red_dragon:0 "Dragon Prince"
+ dragonprincess_female_red_dragon:0 "Dragon Princess"
+
+ ### Mantids ###
+ emperor_male_mantid:0 "Grand Emperor"
+ empress_female_mantid:0 "Grand Empress"
+
+ imperial_vizier_male_mantid:0 "Imperial Vizier"
+ imperial_vizier_female_mantid:0 "Imperial Vizier"
+ grand_vizier_male_mantid:0 "Grand Vizier"
+ grand_vizier_female_mantid:0 "Grand Vizier"
+ vizier_male_mantid:0 "Vizier"
+ vizier_female_mantid:0 "Vizier"
+
+ ### Qiraji ###
+ # Special
+ emperor_ahn_qiraj_male_qiraji:0 "Emperor"
+ empress_ahn_qiraj_female_qiraji:0 "Emperor"
+ empire_ahn_qiraj:0 "Empire"
+
+ # Others
+ lord_male_qiraji:0 "Lord"
+ lady_female_qiraji:0 "Lady"
+ hive_qiraji:0 "Hive"
+ high_overlord_male_qiraji:0 "High Overlord"
+ high_overlady_female_qiraji:0 "High Overlady"
+ brood_qiraji:0 "Brood"
+
+ ### Wildhammer ###
+ # Feudal
+ chief_male_wildhammer:0 "Chief"
+ chieftess_female_wildhammer:0 "Chieftess"
+ chiefdom_wildhammer:0 "Chiefdom"
+ thane_male_wildhammer:0 "Thane"
+ thane_female_wildhammer:0 "Thane"
+ high_chiefdom_wildhammer:0 "High Chiefdom"
+ high_thane_male_wildhammer:0 "High Thane"
+ high_thane_female_wildhammer:0 "High Thane"
+
+ ### Dark Iron ###
+ # Feudal
+ lord_male_darkiron:0 "Lord"
+ lady_female_darkiron:1 "Lady"
+ lordship_darkiron:0 "Lordship"
+ thane_male_darkiron:0 "Thane"
+ thane_female_darkiron:0 "Thane"
+
+ ### Old Dwarven / Frostborn ###
+ # Feudal
+ chief_male_old_dwarven:0 "Chief"
+ chieftess_female_old_dwarven:0 "Chieftess"
+ chiefdom_old_dwarven:0 "Chiefdom"
+ thane_male_old_dwarven:0 "Thane"
+ thane_female_old_dwarven:0 "Thane"
+ high_chiefdom_old_dwarven:0 "High Chiefdom"
+ high_thane_male_old_dwarven:0 "High Thane"
+ high_thane_female_old_dwarven:0 "High Thane"
+ high_king_male_old_dwarven:0 "High King"
+ high_queen_female_old_dwarven:0 "High Queen"
+ high_kingdom_old_dwarven:0 "High Kingdom"
+
+ ### Vrykul and Kvaldir ###
+ # Feudal
+ thane_male_vrykul:0 "Thane"
+ thane_female_vrykul:0 "Thane"
+ chiefdom_vrykul:0 "Chiefdom"
+ jarl_male_vrykul:0 "Jarl"
+ frue_female_vrykul:0 "Frue"
+ jarldom_vrykul:0 "Jarldom"
+ high_king_male_vrykul:0 "High King"
```

Comment:
Should just be King (https://wowpedia.fandom.com/wiki/King_Ymiron).

Reply:
The royal title is always "King", the imperial title is "High King" as for the Norse in vanilla and as for the humans in the mod.
Maybe i add the tier to the syntaxt to make it less confusing ?